### ISSUE TITLE
Plugin: Fix reserved keyword variable names phpcs reports

### ISSUE THREAD

**Issue by reykjalin** [Automattic/woocommerce-payments#8543]
Part of #8438.

## Description

Remove all the `Universal.NamingConventions.NoReservedKeywordParameterNames` excludes from `phpcs.xml.dist` and fix the issues reported when running `npm run lint:php` in the following files:

- `tests/unit/helpers/class-wc-mock-wc-data-store.php`
- `includes/wc-payment-api/class-wc-payments-api-client.php`
- `tests/unit/wc-payment-api/test-class-wc-payments-api-client.php`
- `includes/class-wc-payments-utils.php`
- `tests/unit/test-class-wc-payments-features.php`
- `src/Internal/Service/SessionService.php`
- `includes/class-wc-payments-webhook-processing-service.php`

**Comment by BurakParsAydin** [comment]
@reykjalin When I run the following commands in order, nothing happens. 

`npm run lint:php`
`./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep '.php$') --sniffs='Universal.NamingConventions.NoReservedKeywordParameterNames'`

I have tried different combinations, but without success. Could you kindly tell me what I might be missing?

![Image](https://github.com/user-attachments/assets/8e7380a8-92f5-4eb5-81dd-1af9271c6fe5)

**Comment by reykjalin** [comment]
Did you remove the hard-coded ignore from the phpcs config?

**Comment by BurakParsAydin** [comment]
@reykjalin No, I haven't done that. I wasn’t aware that there is an ignore rule in the phpcs file. I would appreciate any further details on what exactly needs to be removed.

As a side note, I have recently started contributing to this repository and am still becoming familiar with the codebase, so I am in the process of learning. 
Thank you for your guidance!

**Comment by reykjalin** [comment]
https://github.com/Automattic/woocommerce-payments/blob/f0b5926bace705fc632d035707075c97ceabd1a3/phpcs.xml.dist#L50-L60

**Comment by BurakParsAydin** [comment]
@reykjalin FYI, I’ve created a PR.

**Comment by reykjalin** [comment]
I don’t work on this anymore, let’s see if @bborman22 can help get looking at the PR prioritized
